reading_json.py

```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import json
+import sys
+
+
+# FINISH TESTING & DEBUGGING
+#
+#
+
+def main():
+
+    # getting cmd_line_args in format:
+    # "python3 reading_json.py file.json img_dir output_dir"
+    cmd_line_args = sys.argv
+    print(cmd_line_args)
+    num_args = len(cmd_line_args)
+
+    # check for correct num of args (expecting 4 args: reading_json.py, file.json, img dir, output_dir)
+    if (num_args != 4):
+        # caller has wrong num of args
+        print("ERR: Arguments entered incorrectly. Please call with the command: 'python3 reading_json.py file.json img_dir output_dir' ")
+        return False
+
+    print("copying images from " + cmd_line_args[2])
+
+    # read input JSON file
+    j_file = (open(cmd_line_args[2])).read()
+
+    # parse JSON into a dictionary:
+    json_dict = json.loads(j_file)
```

Comment:
Thanks for not putting `(open(cmd_line_args[2])).read()` into this as well! :) While its good to save lines of code, its also important to remember that us programmers have to read through it too! Good job